static/js/containers/Payment.js

```diff
@@ -100,55 +101,74 @@ class Payment extends React.Component {
       {defaultValue: ""};
 
     return (
-      <select
-        className="klass-select"
-        onChange={this.setSelectedKlassIndex}
-        {...valueProp}
-      >
-        {options}
-      </select>
+      <div className="klass-select-section">
+        <p className="desc">
+          Which Bootcamp do you want to pay for?
+        </p>
+        <div className="styled-select-container">
+          <select
+            className="klass-select"
+            onChange={this.setSelectedKlassIndex}
+            {...valueProp}
+          >
+            {options}
+          </select>
+        </div>
+      </div>
     );
   };
 
-  renderSelectedKlass() {
+  renderSelectedKlass = () => {
     const {
       ui: { paymentAmount },
       payment: { processing },
       selectedKlass
     } = this.props;
 
-    return <div>
-      <h2 className="klass-title">{selectedKlass.klass_name}</h2>
-      <span>You have paid ${selectedKlass.total_paid} out of ${selectedKlass.price}</span>
+    let deadlineDateText;
+    if (!_.isEmpty(selectedKlass.payment_deadline)) {
+      let deadlineDate = moment(selectedKlass.payment_deadline).format("MMM D, YYYY");
+      deadlineDateText = `You can pay any amount, but the full payment must be complete by ${deadlineDate}`;
+    } else {
+      deadlineDateText = 'You can pay any amount toward the total cost.';
```

Comment:
The two lines above are now covered but this line is now uncovered. Can you add a test for this case?